tls-client/main.cpp

```diff
@@ -50,7 +50,7 @@
 
 namespace {
 
-const char *HTTPS_SERVER_NAME = "developer.mbed.org";
+const char *HTTPS_SERVER_NAME = "os.mbed.org";
```

Comment:
On second thought, it seems that the domain should actually be `os.mbed.com`. I tried `os.mbed.org` and that says the site is unreachable

Reply:
Oops my bad , updated the PR.